src/main/java/com/pusher/client/crypto/nacl/SecretBoxOpener.java

```diff
@@ -26,8 +26,7 @@ a copy of this software and associated documentation files (the "Software"),
 
 import static com.pusher.client.util.internal.Preconditions.checkArgument;
 import static com.pusher.client.util.internal.Preconditions.checkNotNull;
-
-import java.util.Arrays;
+import static java.util.Arrays.fill;
```

Comment:
I'm not the biggest fan of static imports, but why not do the same for `System.arraycopy`?

Reply:
I've considered using static import for `System.arraycopy` but it feels like it's good to convey the information that it's about using quick native OS ("System") call to copy a continuous part of the memory as opposed to Java implementation iterating over all elements.

With `fill` the call site is as legible without `Arrays.` which was just the noise.

Out of curiosity why are you not a fan of static imports? You didn't seem to have problems with top level functions in Kotlin which are a very similar thing, indeed on jvm they are the same.